src/transformers/models/llama/modeling_llama.py

```diff
@@ -802,6 +808,9 @@ def _init_weights(self, module):
             more detail.
         return_dict (`bool`, *optional*):
             Whether or not to return a [`~utils.ModelOutput`] instead of a plain tuple.
+        use_legacy_cache (`bool`, *optional*):
+            If set to `True` (default), will return `past_key_values` as described input above. Otherwise, will return
+            a subclass of `Cache`
```

Comment:
```suggestion
        use_legacy_cache (`bool`, *optional*):
            If set to `True` (default), will return `past_key_values` as a tuple, which is what we call the `legacy` cache format. Otherwise, will return a subclass of [`~cache_utils.Cache`]
```